raiden/api/python.py

```diff
@@ -1856,6 +1858,10 @@ def create_light_client_payment(
 
         if channel_state:
 
+            # checking the balance before creating the payment
+            if amount > get_distributable(channel_state.our_state, channel_state.partner_state):
+                raise InsufficientFunds("Insufficient funds to create payment")
```

Comment:
are you sure we need this text given the exception type?

Reply:
Yes because if you don't put text there the errors array at the api response will be empty